src/sas/qtgui/Perspectives/Fitting/ReportPageLogic.py

```diff
@@ -10,8 +10,12 @@
 import urllib.parse
 
 from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+from bumps import options
 
 import sas.qtgui.Utilities.GuiUtils as GuiUtils
+from sas.sasview import __version__ as SASVIEW_VERSION
+from sasmodels import __version__ as SASMODELS_VERSION
```

Comment:
I would probably pedantically change the import order and put `sasmodels` after `bumps`, since `sasmodels` are higher in hierarchy than `sas`, on level with `bumps` but this is just a general comment.